### ISSUE TITLE
On MacOS, View menu has Enter Full Screen added to it

### ISSUE THREAD

**Issue by scottnonnenberg** [nwjs/nw.js#6332]
NWJS Version : `0.27.1-sdk`
Operating System : MacOS 10.11.6

### Expected behavior

My View menu should not have an 'Enter Full Screen' item added to it

### Actual behavior

My View menu has an 'Enter Full Screen' item added at the bottom

### How to reproduce

Extract the attached file and run `nw .` in the same directory as the `package.json`. You'll note that the View menu, which just had one item 'View Action 1' added to it, now has two elements.

[Archive.zip](https://github.com/nwjs/nw.js/files/1564300/Archive.zip)

Seems there's a specific MacOS API to prevent this behavior: https://github.com/electron/electron/issues/3038#issuecomment-162256354

**Comment by Christywl** [comment]
I can reproduce this issue on Mac with nwjs-sdk-v0.27.2. 
Not reproduce on Linux/Windows.

**Comment by gpetrov** [comment]
Yes this is very annoying we could get rid of the default "Enter Full Screen" menu entry either... we just wanted to have a custom one.

**Comment by toxygen** [comment]
Confirming this issue, very annoying. Just using different submenu name causes different behavior - I don't think this is neither documented nor expected.

<img width="253" alt="Screenshot 2022-10-15 at 21 55 23" src="https://user-images.githubusercontent.com/287405/196005485-aa4699ff-039c-4c20-a64c-e28b630efcbd.png">
<img width="264" alt="Screenshot 2022-10-15 at 21 58 42" src="https://user-images.githubusercontent.com/287405/196005488-db0e39f4-0aa7-405c-b916-a77edea05117.png">
